tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2194,8 +2194,7 @@ public void handlePath(String xpath) {
             // Also ldml.dtd is required -- and should already have been created by ST when generating vxml
             final private String vxmlDir = "../vetdata-2023-01-23-plain-dropfalse/vxml/";
             final private File[] list = new File[]{
-                new File(vxmlDir + "common/main/"),
-                new File(vxmlDir + "common/annotations/")
+                new File(vxmlDir + "common/" + new File(options[SOURCEDIR].value).getName())
```

Comment:
@btangmu this doesn't work! options aren't parsed when this object is created.

Reply:
Filed https://unicode-org.atlassian.net/browse/CLDR-16343 for cleaning up CLDRModify tech debt, and worked around this in this PR.